wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1711,7 +1705,7 @@ def other():
 @final
 class UnhashableTypeInHashViolation(ASTViolation):
     """
-    Forbids to use exlicit unhashable types as set items and dict keys.
+    Forbids to use explicit unhashable types of asset items and dict keys.
```

Comment:
Think that should still say "as set".
i.e. list cannot be used as a set item (an item in a set).